test/embed.test.js

```diff
@@ -51,20 +60,25 @@ describe('Embed Tests', () => {
     },
   });
 
-  beforeEach(function beforeEach() {
-    this.polly.configure({
-      matchRequestsBy: {
-        headers: {
-          exclude: ['user-agent', 'accept'],
-        },
-      },
+  beforeEach(function test() {   
+    this.polly.server.any().on('beforePersist', (req, recording) => {
+      // this is really missing in pollyjs!
+      if (recording.response.cookies.length > 0){
+        recording.response.cookies = [];
+      }
+
+      Object.entries(recording.response.headers).forEach(([k, v]) => {
+        if (v.name === 'set-cookie'){
+          recording.response.headers[parseInt(k)] = {};
```

Comment:
why a `parseInt()` ?

Reply:
I appreciate the reviews! Thanks @tripodsan hmmm because the headers look like

"0": {"name": "set-cookie", "value": "blahblah"}
"1": {........}

Reply:
yes, but if `response.headers` is an object, with a string  value as index, then why do you need to convert the `k` to a number again? it's not an array.

Reply:
Are you sure it's not an Array? That would make things a lot easier.

Reply:
^^ my blunder; yup you guys are totally right here! 
fixed!